quarkus.jdt.ext/com.redhat.microprofile.jdt.quarkus/src/main/java/com/redhat/microprofile/jdt/internal/quarkus/providers/QuarkusStaticConfigSourceProvider.java

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+* Copyright (c) 2021 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.microprofile.jdt.internal.quarkus.providers;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.jdt.core.IJavaProject;
+import org.eclipse.lsp4mp.jdt.core.project.IConfigSource;
+import org.eclipse.lsp4mp.jdt.core.project.IConfigSourceProvider;
+import org.eclipse.lsp4mp.jdt.core.project.PropertiesConfigSource;
+
+/**
+ * Provides configuration sources specific to Quarkus.
+ *
+ * <ul>
+ * <li><code>application.properties</code></li>
+ * <li><code>application.yaml</code></li>
+ * <li><code>application.yml</code></li>
+ * </ul>
+ *
+ */
+public class QuarkusStaticConfigSourceProvider implements IConfigSourceProvider {
```

Comment:
Why `QuarkusStaticConfigSourceProvider` and not `QuarkusConfigSourceProvider` ? perhaps it's because you wouldlike to manage profile like `dev.application.properties`? If it that we could manage that in the same `QuarkusConfigSourceProvider` class.

Reply:
Okay. That was my plan. I will rename into `QuarkusConfigSourceProvider`